BackEndFlask/controller/Routes/Rating_routes.py

```diff
@@ -10,24 +16,31 @@ def get_student_individual_ratings():
     # and their CompletedAssessmentTasks
     assessment_task_id = int(request.args.get("assessment_task_id"))
     if assessment_task_id == type(""):
-        print(f"[ Rating /rating GET] An error occurred retrieving all ratings for assessmet_task_id: {assessment_task_id}")
+        print(f"[ Rating /rating GET] An error occurred retrieving all ratings for assessment_task_id: {assessment_task_id}")
         createBadResponse(f"An error occurred retrieving all ratings for assessment_task_id: {assessment_task_id}!")
         return response
     student_completed_assessment_tasks =  get_individual_completed_and_student(assessment_task_id)
-    if student_completed_assessment_tasks == type(""):
-        print(f"[ Rating /rating GET] An error occurred retrieving all ratings for assessmet_task_id: {assessment_task_id}")
+    feedback = student_completed_assessment_tasks[0][3]
+    submission = student_completed_assessment_tasks[0][4]
+    feedback_id = student_completed_assessment_tasks[0][6]
+    lag_time = feedback - submission
+    update_lag_time(lag_time, feedback_id)
+    student_completed_assessment_tasks_with_lag_time =  get_individual_completed_and_student(assessment_task_id)
```

Comment:
There is no reason to do this a whole other request to the database here. All of the information it will return is already in memory and could be placed into a dictionary before being passed to Marshmallow to be dumped.

Reply:
Additionally, the seconds problem could be avoided by converting lag time to a string before passing to marshmallow using `.strftime()`.